---
title: "Discuss the principle behind nanopore sequencing and its application in detecting methylated bases. "
tags:
 - Genes, Genomes, and Chromosomes
---
Nanopore sequencing is a novel third generation sequencing technology developed by Oxford Nanopore, a spin-out company from the University of Oxford in the United Kingdom. 

This technology works by passing nucleic acid through a pore protein (CsgG) embedded in an artificial, electrically insulating, membrane. The polynucleotide is pushed through with a motor protein, which although isnâ€™t publicly known, is likely to be a helicase. 
The flow of electrons around the nucleic acid in the pore protein is recorded, and this can be converted to a sequence by computational analysis. This provides a real-time sequence, removing the assembly steps required for shotgun techniques like Illumina sequencing (a next generation sequencing technology), giving a final sequence more quickly. 
Nanopore sequencing gives longer read lengths, of up to 2 megabases. This is considerably longer than other technologies, with Illumina giving read lengths of about 150 bases, Sanger sequencing having 800 base reads, and PacBio reaching around 50 kilobases. Due to the long reads, discontinuous sequences are less likely to be output. 

As the nanopore is able to have methylated bases pass through, the electron signature for these is able to be identified and converted, providing further information about transcriptional activity (especially in promoter regions and CpG islands). This provides additional data without requiring any library preparation processes. 
